.github/workflows/nri-config-generator.yml

```diff
@@ -12,6 +12,9 @@ on:
   pull_request:
     branches: [ main ]
     paths: [ nri-config-generator/** ]
+  push:
```

Comment:
should we add the security jobs ?

Reply:
Since we are not including `rt` (at least we are not including it yet) I didn't consider the security job, but we could do it as part of this change 🤔 WDYT?

Reply:
Perhaps only for the generator that gets automerged deps ?

Reply:
Security scan included here: [3b6b5fd](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/187/commits/3b6b5fdbbda10580b24040ada1200a763600c7f4)